src/gui/entry/EditEntryWidget.cpp

```diff
@@ -281,6 +281,8 @@ void EditEntryWidget::setupSSHAgent()
     connect(m_sshAgentUi->decryptButton, SIGNAL(clicked()), SLOT(decryptPrivateKey()));
     connect(m_sshAgentUi->copyToClipboardButton, SIGNAL(clicked()), SLOT(copyPublicKey()));
 
+    connect(m_advancedUi->attachmentsWidget->entryAttachments(), SIGNAL(modified()), SLOT(updateAttachments()));
```

Comment:
updateAttachments() should be updateSSHAgentAttachments()? I got the following error on the latest git:
```
QObject::connect: No such slot EditEntryWidget::updateAttachments()
QObject::connect:  (receiver name: 'EditWidget')
QObject::connect: No such slot EditEntryWidget::updateAttachments()
QObject::connect:  (receiver name: 'EditWidget')
```

Reply:
Confirmed and fixed. Must have forgotten to add the final version to the commit. Now works and does not give the error message and updates as expected.